azure/template.json

```diff
@@ -11,6 +11,10 @@
         "serviceName": {
             "type": "string"
         },
+        "certificateName": {
+            "type": "string",
+            "defaultValue": ""
+        },
         "configNames": {
             "type": "string",
             "defaultValue": "__SolutionBaseName__"
```

Comment:
This needs to be changed to the solution name

Reply:
@adam230594  corrected it